ganache_cli.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 
-ganache-cli -l 80000000 -i 42 --db './ganacheDB' \
---account="0x00f49bc12b2e102e072ee086482e700ad7fe2b5ee417697b13ca04e4dc1572d9,10000000000000000000000000"
\ No newline at end of file
+ganache-cli -l 80000000 -i 42 \
+--account="0x00f49bc12b2e102e072ee086482e700ad7fe2b5ee417697b13ca04e4dc1572d9,10000000000000000000000000" \
+--db './ganacheDB'
```

Comment:
When running this, I get `Error: ENOENT: no such file or directory, open 'ganacheDB/!blockHashes!0x8b02e4a1d0666e20b88b5fabce4650a62353991694cc5c1e28b340f022ef5d2c'`

Reply:
hmmm probably because the current script isn't designed to run with persistence. lemme make them both compatible.

Reply:
ok should be fixed now @michielderoos: if you launch with the quick setup script, shut down and then start up with this script, everything should behave,  though the private key does need to be sent in the env variable